resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -63,64 +63,105 @@ Item
         // the cloud contents area.
         Column
         {
-            anchors.centerIn: parent
+            anchors.horizontalCenter: parent.horizontalCenter
             width: parent.width
             height: childrenRect.height
 
             spacing: 20 * screenScaleFactor
 
-            Image  // Cloud image
+            // Cloud image
+            Image
             {
                 id: cloudImage
                 anchors.horizontalCenter: parent.horizontalCenter
                 source: UM.Theme.getImage("first_run_ultimaker_cloud")
+                scale: 0.666
             }
 
-            Label  // A title-ish text
+            // Motivational icons
+            Row
             {
-                id: highlightTextLabel
-                anchors.horizontalCenter: parent.horizontalCenter
-                horizontalAlignment: Text.AlignHCenter
-                text: catalog.i18nc("@text", "Your key to connected 3D printing")
-                textFormat: Text.RichText
-                color: UM.Theme.getColor("primary")
-                font: UM.Theme.getFont("medium")
-                renderType: Text.NativeRendering
-            }
+                id: motivationRow
+                width: parent.width
 
-            Label  // A number of text items
-            {
-                id: textLabel
-                anchors.horizontalCenter: parent.horizontalCenter
-                text:
+                Column
                 {
-                    // There are 3 text items, each of which is translated separately as a single piece of text.
-                    var full_text = ""
-                    var t = ""
-
-                    t = catalog.i18nc("@text", "- Customize your experience with more print profiles and plugins")
-                    full_text += "<p>" + t + "</p>"
-
-                    t = catalog.i18nc("@text", "- Stay flexible by syncing your setup and loading it anywhere")
-                    full_text += "<p>" + t + "</p>"
+                    id: marketplaceColumn
+                    width: Math.round(parent.width / 3)
+                    spacing: UM.Theme.getSize("default_margin").height
+
+                    Image
+                    {
+                        id: marketplaceImage
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        source: UM.Theme.getIcon("plugin_48px")
```

Comment:
This image won't scale with the pixel scaling factor right now, so it will appear out of proportion for people on screens with different DPI.

It needs to either:
* use a size from the theme, which automatically scales,
* calculate its size from something that scales, such as line 90 `parent.width / 3`.
* explicitly multiply a hard-coded size with the `screenScaleFactor` variable.

Also, the width of the image currently depends on the source image which is bad practice. It would be more flexible in our design to give this image an explicit width.

I'm thinking of something along the line of `width: 48 * screenScaleFactor` but it should be considered if this needs to be a consistent size across the whole interface of Cura, i.e. a theme entry.